docs/source/debugging.rst

```diff
@@ -189,9 +190,38 @@ Defaults currently look like the following:
      distributed.client: warning
      bokeh: error
 
-So, for example, you could add a line like ``distributed.worker: debug`` to get
+The specific components which you can set are ``distributed.client``,  ``distributed.scheduler``,
+``distributed.nanny``, and ``distributed.worker`` so, for example, you could add a line
+like ``distributed.worker: debug`` to get
```

Comment:
I suspect these are probably the most widely configured logs, but in general other modules like `distributed.core` or `distributed.utils` also have loggers that can be configured. Could we add an "etc." to the existing list to leave the door open for other loggers

```suggestion
Logging for specific components like ``distributed.client``,  ``distributed.scheduler``,
``distributed.nanny``, ``distributed.worker``, etc. can each be independently configured. So, for example, you could add a line like ``distributed.worker: debug`` to get
```